pilot/pkg/serviceregistry/kube/conversion.go

```diff
@@ -233,3 +236,29 @@ func PodTLSMode(pod *corev1.Pod) string {
 	}
 	return model.GetTLSModeFromEndpointLabels(pod.Labels)
 }
+
+// IsAutoPassthrough determines if a listener should use auto passthrough mode. This is used for
+// multi-network. In the Istio API, this is an explicit tls.Mode. However, this mode is not part of
```

Comment:
It is used for multi-network to automatically expose all mesh services on the gateway.

Which is a security risk and a major problem in the multi-network design, and quite inconsistent with  the new Gateway API.

I am quite uncomfortable to this - I was hoping we either start using HBone for multi-network ( so there is at least some authentication ) or explicit opt in for services to be exposed in the E/W.

Note that Gateway API does NOT send traffic even to multiple clusters in the same network - there is an explicit opt-on via backendRef of ServiceImport kind.

Reply:
Don't we have opt-in via `servers.hosts`? Not as nice as  an attached backendRef, but HTTPRoute won't make sense for TCP workloads.